use-case-guides/workday-request-time-off/README.md

```diff
@@ -208,150 +183,330 @@ curl --location 'https://{{domain}}.workday.com/ccx/api/absenceManagement/v1/{{i
+    * Connector: Use your iPaaS connector here
+    * API Endpoint Path: `/workers/{{user.email_addr}}/time-off`
+    * Method: `POST`
+    * Headers: `Content-Type` : `application/json`
+    * Query Params: Leave Blank
+    * Request Body: Use the below JSON to configure your API action. The time off type id can be found from running **API 2** for an employee's email.
+    ```json
+    {
+        "start_date": "{{start_date}}",
+        "time_off_type_id": "abc1234"
     }
+    ```
+
+    ![Submit Time Off](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-9.png)
+
+11. If your employee selects yes, ask for the end date and additional comments for their time off request.
+    * End Date: this is configured as a Date slot
+    * Comments: this is configured as a Free Text input
 
-    to_type_id = CANONICAL_TIME_OFF_PLAN_MAP.get(totype, "")
-    if not to_type_id:
-        return {"error": "Invalid time-off type provided."}
-    
-    payload = {
-        "start_date": start_date,
-        "end_date": end_date or start_date,
-        "type": to_type_id,
-        "comments": comments,
-        "start_time": start_time,
-        "end_time": end_time,
+    ![Select Other Options](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-10.png)
+
+12. Configure an API action to submit your time off request if your employee selects yes.
+    * If you use the iPaaS code below, configure your API action as follows:
+    * Connector: Use your iPaaS connector here
+    * API Endpoint Path: `/workers/{{user.email_addr}}/time-off`
+    * Method: `POST`
+    * Headers: `Content-Type` : `application/json`
+    * Query Params: Leave Blank
+    * Request Body: Use the below JSON to configure your API action. The time off type id can be found from running **API 2** for an employee's email.
+    ```json
+    {
+        "start_date": "{{start_date}}",
+        "time_off_type_id": "{{time_off_type_id}}",
+        "end_date": "{{end_date}}",
+        "comment": "{{comments}}"
     }
-    
-    response = requests.post(url_request_off, json=payload, headers=headers)
-    if response.status_code == 200:
-        return {"message": "Time off requested successfully."}
-    else:
-        return {"error": f"Request failed with status code {response.status_code}: {response.text}"}
-```
+    ```
+    ![API Action](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-11.png)
 
 # Congratulations!
 
-You just gave your employees the ability to take time off through your Copilot 🏝️ Look at our other Workday use cases below to further improve their experience below, and get inspired on what to build next.
+You just gave your employees the ability to take time off through your Copilot 🏝️ Look at our other Workday plugins below to further improve their experience below, and get inspired on what to build next.
```

Comment:
The emoji placement here makes for a weird read: 
![CleanShot 2024-08-08 at 02 03 56@2x](https://github.com/user-attachments/assets/8ea7dd5f-e43c-4a9b-9e28-c11d6ae7839c)

Reply:
removed